CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Mapbox welcomes participation and contributions from everyone.
 
 #### Bug fixes and improvements
 - Fixed reroute request interruption when setting the `NavigationRerouteController` [#5950](https://github.com/mapbox/mapbox-navigation-android/pull/5950).
+- Made `rerouteController` argument in `MapboxNavigation#setRerouteController` nullable. Null can be passed to disable automatic rerouting [#5957](https://github.com/mapbox/mapbox-navigation-android/pull/5957).
```

Comment:
There should be a link to this PR, not to the issue it fixes. Also dots should be placed before the link, but I see that not everyone respects this rule. Take a look at https://github.com/mapbox/navigation-sdks/blob/main/documentation/android-changelog-guidelines.md. 
```suggestion
- Fixed reroute request interruption when setting the `NavigationRerouteController`. [#5950](https://github.com/mapbox/mapbox-navigation-android/pull/5950)
- Made `rerouteController` argument in `MapboxNavigation#setRerouteController` nullable. Null can be passed to disable automatic rerouting. [#5977](https://github.com/mapbox/mapbox-navigation-android/pull/5977)
```

Reply:
> Also dots should be placed before the link, but I see that not everyone respects this rule.

If you're speaking about the previous line, that was me. :)